src/test/storage/simd_bp128_test.cpp

```diff
@@ -82,6 +82,43 @@ TEST_P(SimdBp128Test, DecompressSequenceUsingIterators) {
   }
 }
 
+// Test that the iterator's advance() method works as expected. Creating a sufficiently large vector to ensure that
+// accesses span multiple bounds. Note, as of February 2020, an issue with proxy iterators being only InputOperators
+// leads to problems when using stl methods such as std::advance.
```

Comment:
"an issue leading to problems" doesn't really help anyone reading this

Reply:
```
// Test that the iterator's advance() method works as expected. Creating a sufficiently large vector to ensure that
// accesses span multiple bounds. Note, as of February 2020, an issue with proxy iterators being only InputOperators
// leads to the problem that STL functionality (e.g., std::advance) does not use the iterator's advance() method to
// move n steps but rather calls increment()/decrement() n times.
```

Better?

Reply:
Yepp. Preferrably, we'd move that text into an issue and link it.

Reply:
Issue created and referenced.

Reply:
Merge issues

Reply:
Done.